baseplate/frameworks/pyramid/__init__.py

```diff
@@ -412,6 +412,7 @@ def includeme(self, config: Configurator) -> None:
         config.set_request_factory(RequestFactory(self.baseplate))
         config.add_subscriber(self._on_new_request, pyramid.events.ContextFound)
         config.add_subscriber(self._on_application_created, pyramid.events.ApplicationCreated)
+        config.add_not_found_view(not_found_override)
```

Comment:
This fails to run, it's `add_notfound_view`. I'll have a fix up with a test in a sec

Reply:
:facepalm: that what I get for reading the doc in markdown and not humanese.